src/core/demangle_cxx.d

```diff
@@ -0,0 +1,152 @@
+module core.demangle_cxx;
+
+version (OSX)
+    enum libCXXABI = "libc++abi.dylib";
```

Comment:
I'm not sure when this library became available. We support macOS back to Lion (10.7).

Reply:
how about i try these in order until a match is found?
```
"libc++abi.dylib"
"libc++.dylib"
"libstdc++.dylib"
```
?

Reply:
Perhaps,  I don't know what is the best solution.

Reply:
done, improved logic to make it work on a best effort basis on osx and linux; trivial to add new search locations in new design